### PR TITLE
Emsdk-848: tzset cache fix

### PR DESCRIPTION
**Pull request by DConmy**
For issue: https://github.com/emscripten-core/emsdk/issues/848

### CODE REVIEW

**Comment by DConmy** [comment]
I will review and address the failures.   Clearly, the failures are around what I had changed.

File: src/library.js [hunk 1]
```diff
@@ -706,16 +706,21 @@ LibraryManager.library = {
 
   // TODO: Initialize these to defaults on startup from system settings.
   // Note: glibc has one fewer underscore for all of these. Also used in other related functions (timegm)
-  tzset__proxy: 'sync',
+  mktime__deps: ['tzset_impl'],
```
Comment (sbc100) [inline src/library.js:709]: `mktime__deps` already exists on line 540, maybe update that one instead (good to keep the deps alongside the implementation)... and actually I think deps of mktime are unchanged by this PR aren't they?

**Review by sbc100 — COMMENTED** [review]
lgtm % comment

File: src/library.js (hunk 1 above)
Comment (DConmy) [reply src/library.js]: @sbc100   Sorry, the deps should have been for tzet, not mktime.   I have since corrected.
I'll watch for the checks to succeed.

**Review by sbc100 — APPROVED** [review]

**Pull request merged**